### ISSUE TITLE
Drill Cycle (G81) does not fully retract after drilling

### ISSUE THREAD

**Issue by RJSavell** [buildbotics/bbctrl-firmware#219]
Until issue #217 is corrected I decided to go back to 0.4.5 so perhaps you all have already seen and fixed it but I found the controller was not returning the Z-Axis to the clearance height (the R value) between drilling the holes 

and just cut across the work piece.  See below:

![image](https://user-images.githubusercontent.com/47092573/57744710-968d0e80-7687-11e9-9656-a73af44e0caf.png)

It first cuts hole #1 raises the Z-Axis but not to the specific value of 3 but instead to a height of about -6mm and then moves to hole #2 cutting across the work piece as shown above.  This continues for holes 3 and 4 until I saw what was happening and pressed stop.  I reloaded the file and tried again and it did exactly the same.  I then reset the controller (which reset the offset value if that is related) and this time did not use the probe but set the height manually and it worked as expected.

I did not think before I reset the controller but I think the offset value for the Z-Axis, after setting the height with the probe, was just under 9mm.

I attached the gcode file as well.

[Bottle Holder.txt](https://github.com/buildbotics/bbctrl-firmware/files/3180692/Bottle.Holder.txt)

**Comment by jcoffland** [comment]
Yes, you are right the canned cycles are not respecting offsets.  This will be fixed in v0.4.8.
